src/eureka/S5_lightcurve_fitting/plots_s5.py

```diff
@@ -52,12 +52,12 @@ def plot_fit(lc, model, meta, fitter, isTitle=True):
         model.physeval(interp=meta.interp)
     model_noGP = model.eval(incl_GP=False)
     model_gp = model.GPeval(model_noGP)
-    model_lc = model_noGP+model_gp
+    model_eval = model_noGP+model_gp
```

Comment:
Any particular reason for all the renaming? Or was it me who renamed things in a previous PR while trying to fix multwhite code and you ended up independently solving the problem in a separate branch?

Reply:
There was a bug lower down where we were using the wrong `model` variable and it only showed up in `multwhite`.  I had to make a decision on how to relabel the parameters in a way that was more informative and this was the path I chose.  For consistency, I adopted the same labelling across the entire file.